packages/nextjs/hooks/scaffold-eth/useDeployedContractInfo.ts

```diff
@@ -1,54 +1,47 @@
 import { useEffect, useState } from "react";
+import contracts from "../../generated/hardhat_contracts";
+import { Contract, ContractCodeStatus, ContractName } from "./contract.types";
+import { useIsMounted } from "usehooks-ts";
 import { useProvider } from "wagmi";
-import { getTargetNetwork } from "~~/utils/scaffold-eth";
-
-type GeneratedContractType = {
-  address: string;
-  abi: any[];
-};
+import scaffoldConfig from "~~/scaffold.config";
 
 /**
  * @dev use this hook to get a deployed contract from `yarn deploy` generated files.
  * @param contractName - name of deployed contract
  * @returns {GeneratedContractType | undefined} object containing contract address and abi or undefined if contract is not found
```

Comment:
This `GeneratedContractType` is not there anymore. Should we just remove the return here?